dynawo/sources/Modeler/DataInterface/IIDM/test/TestCriteria.cpp

```diff
@@ -566,8 +566,9 @@ TEST(DataInterfaceIIDMTest, testBusCriteriaDataIIDM) {
   collection->add(CriteriaCollection::BUS, criteria);
   data = createNodeBreakerNetworkCriteria();
   exportStates(data);
+  data->configureCriteria(collection);
   // voltage level not found
```

Comment:
Are we sure that the criterion would  fail if the voltage level was ok?

Reply:
This is the goal of the next 3 tests !
It follows the same principle as the previous tests with a dummy bus then bus ok.